### PR TITLE
PMM-8869 readme update

### PR DESCRIPTION
**Pull request by vyarosh**

### CODE REVIEW

File: docker-compose-allure.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+services:
+  allure:
+    image: "frankescobar/allure-docker-service"
+    environment:
+      CHECK_RESULTS_EVERY_SECONDS: 1
+      KEEP_HISTORY: 1
+    ports:
+      - "5050:5050"
+    volumes:
+      - ./output/allure:/app/allure-results
```
Comment (puneet0191) [inline docker-compose-allure.yml:13]: This should be ./tests/output right?

File: docker-compose-allure.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    volumes:
+      - ./tests/output/allure:/app/allure-results
+      - ./tests/output/allure:/app/default-reports
+  allure-ui:
+    image: "frankescobar/allure-docker-service-ui"
+    environment:
+      ALLURE_DOCKER_PUBLIC_API_URL: "http://localhost:5050"
+      ALLURE_DOCKER_PUBLIC_API_URL_PREFIX: ""
+    ports:
+      - "5252:5252"
```
Comment (yurkovychv) [inline docker-compose-allure.yml:21]: new line pls

File: pr.codecept.js [hunk 3]
```diff
@@ -75,6 +75,8 @@ exports.config = {
   },
   mocha: {
     reporterOptions: {
+      reportDir: './tests/output',
```
Comment (yurkovychv) [inline pr.codecept.js:78]: please test your changes on CI

Comment (vyarosh) [reply pr.codecept.js]: reverted and updated readme.https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/10318/pipeline/

**Review by yurkovychv — APPROVED** [review]
Great job. Thanks

**Pull request merged**